apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/UpcomingInvoice.tsx

```diff
@@ -195,33 +232,39 @@ const UpcomingInvoice = ({ slug }: UpcomingInvoiceProps) => {
                 </Collapsible>
               ))}
 
+            {computeCredits && (
```

Comment:
Thinking about it, or rather looking at it, it does look a little odd that after all these broken down metrics we have one last item without a breakdown. I don't have too hard feelings on this, but wdyt?

Reply:
See my comment in Slack.